Dockerfile

```diff
@@ -1,7 +1,11 @@
-FROM centos/python-27-centos7:latest
+FROM centos:latest
 
 ADD requirements.txt .
 
+RUN yum -y install epel-release && yum clean all
+
+RUN yum -y install python-pip && yum clean all
```

Comment:
we should combine these commands, it will reduce the number of layers in the final image. something like this should work:

```
RUN yum -y install epel-release && \
    yum -y install python-pip && \
    yum clean all
```